test/index.html

```diff
@@ -18,7 +18,10 @@
 <body>
   <script>
     WCT.loadSuites([
-      'lazy-imports_test.html',
+      '1.x/lazy-imports_test.html?dom=shadow',
```

Comment:
don't these tests need to be setup to only run under the version of Polymer they target? I think there might need to be some logic to check `Polymer.version`. Maybe @valdrinkoshi can advise